tfhe/tests/backward_compatibility/shortint.rs

```diff
@@ -0,0 +1,115 @@
+use std::path::Path;
```

Comment:
last question given you want to have a backward module per API layer, should this be moved ?

Reply:
You mean the whole file ? I consider this to be more like integration tests than unit tests, which by cargo conventions reside in `$crate_root/tests`.

Reply:
ok !